web/onos-gui/src/app/onos-config/models-list/models-list.component.ts

```diff
@@ -110,4 +103,22 @@ export class ModelsListComponent extends TableBaseImpl implements OnInit, OnDest
         }
         this.newConfigTitle = '';
     }
+
+    onSortCol(colName: string): void {
+        this.modelService.switchSortCol(colName.toLowerCase(), this.modelService.sortParams.sortDirMap.get(colName));
+        const newDir = this.modelService.sortParams.sortDirMap.get(colName) === 0 ? 1 : 0;
+        this.modelService.sortParams.sortDirMap.set(colName, newDir);
+        this.sortIcon(colName);
+    }
+
+
+    sortIcon(colName: string): string {
+        if (colName === this.modelService.sortParams.firstColName) {
+            if (this.modelService.sortParams.firstCriteriaDir === 0) {
+                return 'downArrow';
+            } else {
+                return 'upArrow';
+            }
```

Comment:
I think there is a 3rd option here - an empty string - for when you do not want to display an icon

Reply:
since there is an additional check for === colName, the empty string is not required